roles/openshift_web_console/defaults/main.yml

```diff
@@ -13,3 +13,16 @@ openshift_web_console_image_name: "{{ l_osm_registry_url | regex_replace('${comp
 
 # Default the replica count to the number of masters.
 openshift_web_console_replica_count: "{{ groups.oo_masters_to_config | length }}"
+
+# openshift_web_console_smoke_test common setup
+openshift_web_console_smoke_test_namespace: openshift-web-console-smoke-test
+openshift_web_console_smoke_test_tmp_location: /tmp
+
+openshift_web_console_smoke_test_pod_template_file: templates/console-smoke-test-pod.yaml.j2
+
+openshift_web_console_smoke_test_service_account: web-console-smoke-test
+openshift_web_console_smoke_test_cluster_role_name: web-console-smoke-test
+
+openshift_web_console_smoke_test_name: openshift-web-console-smoke-test-pod
+openshift_web_console_delete_tempfiles: True
```

Comment:
I don't see a lot of value making most of these configurable. I think it's OK to hard-code the namespace / deployment name, etc. (We do for the console itself.)

We might make test interval configuration, though.